src/style.css

```
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+}
+
+.search_bar {
+  width: 100%;
+  position: relative;
+}
+
+.search_barBig_input {
```

```suggestion
.search_bar--big_input {
```
don't join separate words (use `--` for modifications of the original class (you will learn more about this in the BEM topic))

---

thank you for the correction
I will keep it in mind